src/test/data/JsonSerializableAddressBookTest/duplicateDoctorAddressBook.json

```diff
@@ -28,7 +28,7 @@
       "name" : "John Doe",
       "phone" : "88887777",
       "email" : "prawn@gmail.com",
-      "height" : "1.7",
```

Comment:
May I check if the height RegEx will fail for heights with 1 decimal point?

Reply:
I believe it does; do we want this behaviour?

Reply:
I think its better for it to be 2 decimal place, since height is usually represented that way. We can write logic to add the 2 decimal place even if the user inputs 1 decimal place but I think getting the user to learn to input 1 decimal place would be easier.